app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -186,6 +128,48 @@ def save_inventory_multi(association, hashes, deletes, find_key, child_keys = []
     association.push(new_records)
   end
 
+  def save_inventory_multi_child(rel_type, association, parent, hashes, find_key, child_keys = [], extra_keys = [],
```

Comment:
Hm, can you describe what is the need for a new saving method? I am trying to get rid of this whole module, so I wonder why we need new saving method. :-)